precompiles/bech32/utils_test.go

```diff
@@ -0,0 +1,13 @@
+package bech32_test
+
+import (
+	"github.com/evmos/evmos/v15/precompiles/bech32"
+)
+
+// setupBech32Precompile is a helper function to set up an instance of the Bech32 precompile for
```

Comment:
For?

Reply:
setting up the precompile and adding it to the Suite.